src/logic/operate.js

```diff
@@ -20,5 +20,5 @@ export default function operate(numberOne, numberTwo, operation) {
       return one.div(two).toString();
     }
   }
-  throw Error(`Unknown operation '${operation}'`);
+  throw Error(`Unknown operation performed'${operation}'`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error message in the new hunk has a syntax error. The opening quote before `${operation}` is missing, which could lead to an incorrect error message being displayed. Here's how you can fix it:

```diff
- throw Error(`Unknown operation performed'${operation}'`);
+ throw Error(`Unknown operation performed '${operation}'`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->